src/Workspaces/Core/Portable/Options/OptionKey.cs

```diff
@@ -48,6 +48,20 @@ public override int GetHashCode()
             return hash;
         }
 
+        public override string ToString()
+        {
+            if (Option is null)
```

Comment:
what is current behavior? is it currently show type name, the default .net behavior? in that case, making ToString to custom behavior seems good one to have regardless whether we also have DebuggerDisplay or not.

Reply:
this.Option = option ?? throw new ArgumentNullException(nameof(option));

by the way, it is impossible for "Option" to be null, so I dont think we need this check.

looks like IOption implementer such as Option<T> do override ToString. so this one having it as well seems consistent with other types involved?